mizar/obj/endpoint.py

```diff
@@ -322,6 +344,34 @@ def get_droplet_ip(self):
     def get_droplet_mac(self):
         return self.droplet_mac
 
+    def get_ingress_networkpolicies(self):
+        return self.ingress_networkpolicies
+
+    def get_egress_networkpolicies(self):
+        return self.egress_networkpolicies
+
+    def add_ingress_networkpolicy(self, ingress_networkpolicy_name):
+        self.ingress_networkpolicies.append(ingress_networkpolicy_name)
+        self.ingress_networkpolicies.sort()
+        #TODO update networkpolicy enforcement map ingress
+        self.store.update_ep(self)
+
+    def add_egress_networkpolicy(self, egress_networkpolicy_name):
+        self.egress_networkpolicies.append(egress_networkpolicy_name)
+        self.egress_networkpolicies.sort()
+        #TODO update networkpolicy enforcement map egress
+        self.store.update_ep(self)
```

Comment:
update_ep func should be modified as well

Reply:
Here means endpoint was updated so refresh it into store. store.update_ep itself doesn't need change. Just perform its function.

Reply:
why not merge with set_egress_networkpolicies / set_ingress_networkpolicy

Reply:
use store_update_obj instead please

Reply:
Fixed